### DISCUSSION TITLE
简化评论区相关

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3541] [功能建议 / Ideas]
V2:
![图片](https://user-images.githubusercontent.com/81268244/181866352-18c365b1-aa0b-4b54-9e47-7e23bcc2f9a5.png)
V1:
![图片](https://user-images.githubusercontent.com/81268244/181866354-66cb63d0-d8a6-4937-aa24-76f99272eb6c.png)

感觉V1的样式更柔和顺眼一些，请问是否可以进行更改？颜色是 rgb(153, 162, 170)
作者大大辛苦了！

**Comment by Tinhone** [comment]
<p>（拿出来晒一晒</p>
<p>ε=ε=ε=(~￣▽￣)~</p>

**Comment by Tinhone** [comment]
#3791 
希望这么改没有问题orz

**Reply by the1812** [reply]
应该没问题, 只是合并发版得等等了, 最近太忙了没啥时间搞
